### PR TITLE
Add procedures to manage DEB content

### PR DESCRIPTION
**Pull request by maximiliankolb**
This PR adds procedures to manage content for Debian and Ubuntu. I will add additional procedures at a later stage/with a separate PR.

### CODE REVIEW

File: guides/common/modules/proc_adding-custom-deb-repository-example-for-debian-11.adoc
Comment (lzap) [inline guides/common/modules/proc_adding-custom-deb-repository-example-for-debian-11.adoc:16]: Out of my curiousity, whats the `\` escape for? To avoid rendering it as a link? I would expect asciidoctor not render a verbose text as links.

File: guides/common/modules/proc_adding-custom-rpm-repositories.adoc
Comment (lzap) [inline guides/common/modules/proc_adding-custom-rpm-repositories.adoc:55]: Nitpick: `_My_Architecture_` is quite weird, we don't have that yet. I would probably go just with intel 64bits as that's what 90% of our users use. But fine with this too, it is ready for the (ARM) future :-)

File: guides/common/modules/proc_adding-custom-deb-repository-example-for-debian-11.adoc
Comment (maximiliankolb) [reply guides/common/modules/proc_adding-custom-deb-repository-example-for-debian-11.adoc:16]: Yes, to avoid rendering the link. In mostly unnecessary when you have something like "foreman.example.com/some/path" which is never reachable. In this specific case, you really don't need to browser debian.org at all.

File: guides/common/modules/proc_adding-custom-rpm-repositories.adoc
Comment (maximiliankolb) [reply guides/common/modules/proc_adding-custom-rpm-repositories.adoc:55]: Technically, it is possible to serve content to non-x86 hosts. I feel like this goes along nicely with `My_Org` and `My_Loc`.

File: guides/common/modules/proc_adding-custom-deb-repositories.adoc
Comment (melcorr) [inline guides/common/modules/proc_adding-custom-deb-repositories.adoc:20]: We had a conversation about recommends before. 
I realise that it is hard for you to follow these conventions if others don't. Ideally, we would just say
```suggestion
To keep things easy to manage and to avoid potential performance and network issues during synchronization, create one repository in {Project}.
```

File: guides/common/modules/proc_adding-custom-deb-repositories.adoc
Comment (melcorr) [inline guides/common/modules/proc_adding-custom-deb-repositories.adoc:18]: I don't understand what you're trying to say here if I am honest.

File: guides/common/modules/proc_adding-custom-deb-repositories.adoc
Comment (melcorr) [inline guides/common/modules/proc_adding-custom-deb-repositories.adoc:31]: This sounds quite strange to be honest.
```suggestion
Ensure that you enter both *Releases* and *Components* exactly as they would be in an `/etc/apt/sources.list` file.
```
What do you mean by would be? We generally avoid would/should/could.

File: guides/common/modules/proc_adding-custom-deb-repositories.adoc
Comment (melcorr) [inline guides/common/modules/proc_adding-custom-deb-repositories.adoc:47]: Best to keep wishes for birthday candles.
```suggestion
. Optional: From the *GPG Key* list, select the GPG key if you want to verify the signatures of the `Release` files associated with the Debian repository.
```

File: guides/common/modules/proc_adding-custom-deb-repositories.adoc
Comment (melcorr) [inline guides/common/modules/proc_adding-custom-deb-repositories.adoc:49]: Who is "yourself" ?

File: guides/common/modules/proc_adding-custom-deb-repositories.adoc
Comment (melcorr) [inline guides/common/modules/proc_adding-custom-deb-repositories.adoc:50]: Again here, when I think of "myself", I rarely imagine myself being authenticated with an SSL client key.

File: guides/common/modules/proc_adding-custom-rpm-repositories.adoc
Comment (melcorr) [inline guides/common/modules/proc_adding-custom-rpm-repositories.adoc:9]: Should it be Yum? I am pretty sure that is the standard?

File: guides/common/modules/proc_extracting-gpg-public-key-fingerprints-from-release-files.adoc
Comment (melcorr) [inline guides/common/modules/proc_extracting-gpg-public-key-fingerprints-from-release-files.adoc:4]: DEB looks like an acronym. 

Would Debian not be better? I googled DEB repositories and just got Debian repositories. 
What is the standard?

**Review by melcorr — CHANGES_REQUESTED** [review]
Apologies for the delays with these. 
I tried on Friday but didn't complete. 
I meant to the weekend, but was unwell. 
Here I am, and am sad it is so late. 
I don't have any objections to just merging as is, but I would appreciate if you could consider my feedback. If you don't want to apply it, it is ok.

File: guides/common/modules/proc_adding-custom-deb-repositories.adoc
Comment (maximiliankolb) [reply guides/common/modules/proc_adding-custom-deb-repositories.adoc:20]: Applied.

File: guides/common/modules/proc_adding-custom-deb-repositories.adoc
Comment (maximiliankolb) [reply guides/common/modules/proc_adding-custom-deb-repositories.adoc:18]: So Debian Bullseye is always Debian 11;  but Debian stable is currrently Debian 11, but will be Debian 12 with the next release.

Reworded.

File: guides/common/modules/proc_adding-custom-deb-repositories.adoc
Comment (maximiliankolb) [reply guides/common/modules/proc_adding-custom-deb-repositories.adoc:31]: Applied and reworded ~would be~ to are.

File: guides/common/modules/proc_adding-custom-deb-repositories.adoc
Comment (maximiliankolb) [reply guides/common/modules/proc_adding-custom-deb-repositories.adoc:47]: 🕯️

File: guides/common/modules/proc_adding-custom-deb-repositories.adoc
Comment (maximiliankolb) [reply guides/common/modules/proc_adding-custom-deb-repositories.adoc:49]: Replaced with the instruction from the custom file repositories procedure.

File: guides/common/modules/proc_adding-custom-deb-repositories.adoc
Comment (maximiliankolb) [reply guides/common/modules/proc_adding-custom-deb-repositories.adoc:50]: Replaced.

File: guides/common/modules/proc_adding-custom-rpm-repositories.adoc
Comment (maximiliankolb) [reply guides/common/modules/proc_adding-custom-rpm-repositories.adoc:9]: Should it be `RPM repositories` or `repositories containing RPM packages`?

I've made the change to move away from the default "repository" = "repositories for EL containing rpm packages".

File: guides/common/modules/proc_extracting-gpg-public-key-fingerprints-from-release-files.adoc
Comment (maximiliankolb) [reply guides/common/modules/proc_extracting-gpg-public-key-fingerprints-from-release-files.adoc:4]: I chose `DEB` as a way of including Debian and Ubuntu repositories. Maybe `repositories containing DEB packages` is a good alternative?

Sometimes, people also use `APT`, but that's more of a standard application to download/install packages.

**Comment by maximiliankolb** [comment]
Resolved most issues and rebased to master. Two threads remain open.

File: guides/common/modules/proc_adding-custom-rpm-repositories.adoc
Comment (melcorr) [reply guides/common/modules/proc_adding-custom-rpm-repositories.adoc:9]: Can we not change it for now and we can go and answer your questions at a later stage. I don't want to block this PR but I don't know the answers? If we don't change, and look through this later?

File: guides/common/modules/proc_extracting-gpg-public-key-fingerprints-from-release-files.adoc
Comment (melcorr) [reply guides/common/modules/proc_extracting-gpg-public-key-fingerprints-from-release-files.adoc:4]: Advanced Package Tool = APT - so this is a valid abbreviation.

However, DEB is not an abbreviation... and I don't find it as a standard reference in other Debian/Ubuntu resources. 
I think the DEB looks really bad and misleading, but I will leave it as your call.

File: guides/common/modules/proc_adding-custom-rpm-repositories.adoc
Comment (maximiliankolb) [reply guides/common/modules/proc_adding-custom-rpm-repositories.adoc:9]: Reverted for now.

File: guides/common/modules/proc_extracting-gpg-public-key-fingerprints-from-release-files.adoc
Comment (maximiliankolb) [reply guides/common/modules/proc_extracting-gpg-public-key-fingerprints-from-release-files.adoc:4]: Changed to `APT`.

**Review by melcorr — APPROVED** [review]
Thank you!

**Pull request merged**
